src/java.base/share/classes/java/lang/Double.java

```diff
@@ -32,6 +32,7 @@
 
 import jdk.internal.math.FloatingDecimal;
```

Comment:
Presumably the FloatingDecimal import here and in Float can be removed.

Reply:
`FloatingDecimal` is used in the `parse*(String)` methods